web/src/components/text-input/styles.ts

```diff
@@ -0,0 +1,85 @@
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../tokens'
+import { getColorDefault, getColorHover } from '../checkbox/styles'
+
+export default createUseStyles({
+  textField: {
+    fontFamily: tokens.fontFamily,
+    borderRadius: ({ component }) => {
+      return component === 'input'
+        ? tokens.borderRadiusFormfieldInput
+        : tokens.borderRadiusFormfieldTextarea
+    },
+    borderWidth: tokens.borderWidthFormfield,
+    borderStyle: 'solid',
+    color: tokens.colorTextPrimary,
+    resize: 'none',
+    backgroundColor: ({ isDisabled, isReadOnly }) => {
+      if (isDisabled) return tokens.colorBackgroundSecondary
+      if (isReadOnly) return tokens.colorBackgroundFormfieldReadonly
+      return undefined
+    },
+    cursor: ({ isDisabled }) => {
+      if (isDisabled) return 'not-allowed'
+      return undefined
+    },
+    height: ({ component, maxLength }) => {
+      if (component === 'input') return tokens.sizeFormfieldInput
+      if (maxLength) {
+        return '92px'
```

Comment:
Probably a new token should be added for this? What do you think? Or it's just single time usage?

Reply:
I think this is single use only. We can improve/add one if we have this value again